### ISSUE TITLE
Additional highway shields in Arkansas

### ISSUE THREAD

**Issue by TheInstrumentalist** [osm-americana/openstreetmap-americana#1156]
So I brought these up in a discord server I'm in, and I was referred to bring these up to this board. Some of you may find these interesting...

Private roads in Washington County, Arkansas, mainly for short driveways
![chrome_drqrsAxFWQ](https://github.com/user-attachments/assets/a2b10ee8-6aac-42bc-acb5-66168aea9e67)

Private roads in Johnson County, Arkansas, similar to Washington County
![chrome_tC55FnBBC9](https://github.com/user-attachments/assets/1a11101a-971d-4602-8389-e2e4e7955030)

County roads in Lee County, Arkansas
![chrome_NvsLcErvPF](https://github.com/user-attachments/assets/eae7bc86-efca-43df-a131-77fdba36e6b9)

Marine Fuel Tax (or 900- series) roads in Arkansas
![chrome_qzfLU6AV0n](https://github.com/user-attachments/assets/190258f0-d7b8-4e8d-8347-22abd94e16aa)

Airport roads in Arkansas (any route designated AR 980)
![chrome_j3dOqUouQa](https://github.com/user-attachments/assets/7856b1fd-65f2-4b7e-9fcb-8fe52ee1ad62)

**Comment by 1ec5** [comment]
Private drives in Washington County appear to be tagged as `name_1=County Road *` from TIGER, even though [they aren’t county roads](https://www.arkansasonline.com/news/2018/dec/26/washington-county-planning-to-expand-ro/). Some of Johnson County’s private roads are tagged as `name_1=Private Road *` while others are `name_1=County Road *`. In [changesets 156,548,573] and [156,548,815](https://www.openstreetmap.org/changeset/156548815), I tentatively went with `network=US:AR:Washington:Residential` and `network=US:AR:Johnson:Private`, respectively, for some that I could see in Bing Streetside imagery. If you happen to know of any others without consulting Google Maps (🙅), then please feel free to retag them similarly in OSM.

We have Lee County:

https://github.com/osm-americana/openstreetmap-americana/blob/88c72adf4bc8b697ebe2c264b368b0e4d497ae62/src/js/shield_defs.js#L947-L953

Airport roads have a special carveout:

https://github.com/osm-americana/openstreetmap-americana/blob/88c72adf4bc8b697ebe2c264b368b0e4d497ae62/src/js/shield_defs.js#L903-L909

Marine Fuel Tax roads are covered by #375.

**Comment by claysmalley** [comment]
Good catch, @TheInstrumentalist. If you can find any more examples of private roads in Arkansas with county-assigned numbers, please share them here.

**Comment by StretchLongfellow** [comment]
AR 980 should be fully mapped; I went down a rabbit hole on that a while back. It does have its unique shield as well.
